src/test/resources/jsonld/BT000128754.json

```diff
@@ -29,13 +63,19 @@
         "id" : "https://d-nb.info/gnd/7749153-1",
         "label" : "Gemeinsame Normdatei (GND)"
       },
-      "label" : "1. Fußball-Club Köln 01/07",
+      "label" : "1. FC Köln",
       "gndIdentifier" : "2050419-6"
     }, {
+      "id" : "https://d-nb.info/gnd/4020517-4",
       "type" : [ "SubjectHeading" ],
-      "label" : "Geschichte"
+      "source" : {
+        "id" : "https://d-nb.info/gnd/7749153-1",
+        "label" : "Gemeinsame Normdatei (GND)"
+      },
+      "label" : "Geschichte, Geschichte",
+      "gndIdentifier" : "4020517-4"
     } ],
-    "label" : "1. Fußball-Club Köln 01/07 | Geschichte"
+    "label" : "1. FC Köln | Geschichte, Geschichte"
```

Comment:
The `subject.componentList.label` and `subject.label` values changed and look wrong ("Geschichte, Geschichte")

Reply:
Also rather re: new data.